test/lib/lbt/bundle/AutoSplitter.js

```diff
@@ -16,7 +16,14 @@ function createMockPool(dependencies) {
 			if (name === "c.js") {
 				info.compressedSize = 512;
 			}
-			return {info, buffer: async () => name.padStart(2048, "*")};
+			return {
+				info,
+				buffer: async () => Buffer.from(name.padStart(2048, "*")),
+				getProject: () => undefined,
```

Comment:
Because of this, the newly added code in AutoSplitter is **never** tested:
https://coveralls.io/builds/24765632/source?filename=lib%2Flbt%2Fbundle%2FAutoSplitter.js#L240

Please test it. Why not mock the project configuration here?

Reply:
done